lua/plenary/path.lua

```diff
@@ -229,6 +240,51 @@ function Path:rmdir()
   uv.fs_rmdir(self:absolute())
 end
 
+function Path:touch(opts)
+  opts = opts or {}
+
+  local mode = opts.mode or 420
+  local parents = F.if_nil(opts.parents, false, opts.parents)
+
+  if self:exists() then return end
```

Comment:
nitpick question: does `touch` update the time of the file being saved?

Reply:
You can do that actually pretty easy with uv